### DISCUSSION TITLE
use_shipping_address_for_billing is stored as string while cp panel needs a boolean

### DISCUSSION THREAD

**Discussion by de-raaf-media** [duncanmcclean/simple-commerce#976] [Help] [answered]
Hi there,

I have a hidden field which contains true or false depending on the "use shipping address for billing".

In the JSON data of orders (I store orders as entries) the value is now:

`"use_shipping_address_for_billing": "false"`

In the CP this value needs to be a Boolean. How do I make sure this is stored as a boolean instead of a string?

Regards,
Mark

**Comment by duncanmcclean** [comment]
What does the hidden field look like in your Antlers template?

**Comment by de-raaf-media** [comment]
I use the following inputs depending on the checkbox state (with quotes around the value has the same result). I also tried true false with and without quotes:

```
<input style="visibility: visible; position: relative" type="text" name="use_shipping_address_for_billing" value=1>
<input style="visibility: visible; position: relative" type="text" name="use_shipping_address_for_billing" value=0>
```

**Reply by duncanmcclean** [reply, marked as answer]
Hmm, I'm not sure, I'll need to give it a go myself when I have some time. 

Maybe try `value="on"` / `value="off"`?

**Reply by de-raaf-media** [reply]
Fantastic! It works. Thanks a lot @duncanmcclean!
